raiden/log_config.py

```diff
@@ -249,5 +252,6 @@ def configure_logging(
 
     # set raiden logging level to DEBUG, to be able to intercept all messages,
     # which should then be filtered by the specific filters
-    structlog.get_logger('').setLevel('DEBUG')
-    structlog.get_logger('raiden').setLevel('DEBUG')
+    if not disable_debug_logfile:
```

Comment:
Have you tested this? My guess is that this doesn't work as expected because it doesn't set the loggers levels properly.

Reply:
I tested that we don't log anything in `raiden-debug.log` with this, but you are correct!
I had misunderstood how this logging config operates (which is why I asked for a review).

But now I think I understand and corrected the error in [this](https://github.com/raiden-network/raiden/pull/2024) PR.

Tested it locally both with/without the argument and with changing `--log-config` and the desired behaviour seems to be there.